### PR TITLE
Switch to foundation

### PR DESCRIPTION
**Pull request by vast**
screen shot 2015-06-24 at 17 27 32

### CODE REVIEW

**Comment by vast** [comment]
It's finally here! @rustemginiyatullin, @mikheevdmitry, please have a look.

File: source/layouts/layout.slim [hunk 1]
```diff
@@ -7,7 +7,8 @@ html
     title = current_page.data.title
     meta name="viewport" content="width=device-width, initial-scale=1"
 
-    = stylesheet_link_tag "application"
+    == javascript_include_tag "//cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.2/modernizr.min.js"
```
Comment (timurvafin) [inline source/layouts/layout.slim]: @vast what's the reason to load up modernizr as the external resource?

Comment (vast) [inline source/layouts/layout.slim]: @timurvafin, there is no excuse for that as now modernizr is bundled with `foundation-rails` and can be safely imported into Middleman's sprockets instance. Fixed it in https://github.com/fs/static-base/commit/299caff0821cddf53748d6595086d5589a469829

**Comment by timurvafin** [comment]
:+1:

**Pull request merged**